src/Adapters/Cache/InMemoryCache.js

```diff
@@ -0,0 +1,69 @@
+const DEFAULT_CACHE_TTL = 60 * 1000;
+
+
+export class InMemoryCache {
+  constructor({
+    ttl = DEFAULT_CACHE_TTL
+  }) {
+    this.ttl = ttl;
+    this.cache = Object.create(null);
+  }
+
+  get(key) {
+    let record = this.cache[key];
+    if (record == null) {
+      return null;
+    }
+
+    // Has Record and isnt expired
+    if (isNaN(record.expire) || record.expire >= Date.now()) {
+      return record.value;
```

Comment:
Should return a promise to match other functions / or other functions should return something else than promises for consistency